### ISSUE TITLE
Encode class name deterministically by file name

### ISSUE THREAD

**Issue by MariusOechslein** [rodolfo-cacacho/3d_mai#1]
### Current Behavior

Right now the class indexes are somehow created by increasing a counter: https://github.com/rodolfo-cacacho/3d_mai/blob/7f3280499b847f25cdca624224b8c16124b688a9/notebooks/boundingbox_whole.py#L6-L10
This results in the problem that the yolo model trains with the wrong class names sometimes, since the class indexes sometimes correspond to different classes, but we have to define the classes manually for the yolov8 training:
https://github.com/rodolfo-cacacho/3d_mai/blob/7f3280499b847f25cdca624224b8c16124b688a9/notebooks/yolo_model/config.yaml#L8

### Expected Behavior

Encode the classes in a deterministic way, for example by getting the class name from the file path and encoding it with [LabelEncoder](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html) or some other way of encoding labels. 

### OR (as discussed with Rodolfo)

just create a file containing which class indices correspond to which class. 
Then we could already more easily manually set the class names when training the model in the config file.
